flow/project.py

```diff
@@ -1410,7 +1391,21 @@ def _add_dummy_operation(job):
         if n > 0:
             context['op_counter'].append(('[{} more operations omitted]'.format(n), ''))
 
-        print(template.render(**context), file=file)
+        status_output = template.render(**context)
+        print(status_output, file=file)
+
+        import mistune
```

Comment:
The reason for having imports near where they are used is if the import would be conditional to a factor like a function being called. Having them here doesn't sure much purpose.

Reply:
I agree, these are for my initial trial, I'll move them once I finalize which package to use